spark-operator-api/src/test/java/org/apache/spark/k8s/operator/spec/RestartPolicyTest.java

```diff
@@ -40,8 +43,8 @@ class RestartPolicyTest {
   @Test
   void testAttemptRestartOnState() {
     for (ApplicationStateSummary stateSummary : ApplicationStateSummary.values()) {
-      assertTrue(attemptRestartOnState(RestartPolicy.Always, stateSummary));
-      assertFalse(attemptRestartOnState(RestartPolicy.Never, stateSummary));
+      assertTrue(attemptRestartOnState(Always, stateSummary));
+      assertFalse(attemptRestartOnState(Never, stateSummary));
```

Comment:
BTW, this is a piggy-back for consistency on importing `RestartPolicy.*` in this file.